packages/react-scripts/fixtures/kitchensink/integration/webpack.test.js

```diff
@@ -71,6 +71,22 @@ describe('Integration', () => {
       );
     });
 
+    it('svg component', async () => {
+      const doc = await initDOM('svg-component');
+
+      expect(doc.getElementById('feature-svg-component').textContent).to.equal(
```

Comment:
I realize this test is not terribly useful but I'm not sure what I can check for here. I can try and improve it or just remove it entirely.

Reply:
Maybe we can do `getComputedStyle` here?

Reply:
This is for the SVG component not SVG in CSS so I don't think it will have any style?

Reply:
Oh okay. Then can’t we test that innerHTML contains some SVG tags?

Reply:
We're not getting a real SVG here because we mock it in `fileTransform.js` like so: `ReactComponent: () => ${assetFilename}`. That's why I initially checked the `textContent` for `logo.svg` but it's empty. I think the parent element would contain the filename but I'm not sure.

Reply:
If that works let’s test for that

Reply:
I have no idea if that works (or how to do it) but I'll give it a try. 😀